sportloop/cycloop-web
# feat(devices): connect HRM and display HR in devices

## Description

This is a PoC PR which can be built upon, for implementing device handling in the app. Currently only heart rate monitors can be connected and observed.

## Motivation and Context

First phase of this software will have BLE support on Chrome (and other supported devices) and Heart Rate, Speed, Cadence, Power and Controllable (ANT+ over BLE) devices will be supported. As the first step of this effort this setup should be established.

## How Has This Been Tested?

Heart Rate Monitor support can already be tested via:

1. Navigate to /devices
2. Press on Heart Rate Monitor
3. Upon prompt, select a viable BLE HRM device
4. Wait and observe as state changes and finally HR data shows up

## Types of changes


## Checklist:
